CreateAboutPage

stevenTCL opened the pull request:
1. create About Page, move the contents on Index Page to About
2. UI modification ( in About css)
3. remove “fork me on GitHub” tab on Index and About Page

stevenTCL committed:
CreateAboutPage

1. create About Page, move the contents on Index Page to About
2. UI modification ( in About css)
3. remove “fork me on GitHub” tab on Index and About Page

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,196 @@
+    <script src="js/site.js"></script>
+</head>
+<body>
+   <!--  <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;heigh ...
```
remove this line

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,196 @@
+</head>
+<body>
+   <!--  <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;heigh ...
+    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span> -->
```
remove this line

chautsunman commented on app/index.html:
```diff
@@ -30,8 +30,8 @@
 <body>
-    <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;height:1px ...
-    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span>
+    <!-- <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;heigh ...
```
remove this line

chautsunman commented on app/index.html:
```diff
@@ -30,8 +30,8 @@
-    <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;height:1px ...
-    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span>
+    <!-- <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;heigh ...
+    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span> -->
```
remove this line

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,196 @@
+    <script src="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.0/angular-material.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular-messages.min.js"></script>
+
+    <script src="js/index.js"></script>
```
create a JS file for this page (about.js)

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,196 @@
+        <!-- START THE FEATURETTES -->
+        <div class="row featurette" id="page_team">
+            <div class="col-md-12">
+                <h2 class="featurette-heading">
```
move this <h2> out of the <div class="col-md-12">

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,196 @@
+                    <br>
+                    <span class="text-muted">{{teamName}}</span>
+                </h2>
+                <div class="row border">
```
use `ng-repeat`

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,196 @@
+    <!-- Customized libraries shared by pages -->
+    <script src="js/site.js"></script>
+</head>
+<body>
```
- add an Angular module `ng-app="teamform-about-app"`
- add a `<div>` with `ng-controller="AboutCtrl">` to include all the contents

stevenTCL committed:
Modification to About

1. created a JS file, but don’t know if I did it correctly
2. removed commented lines
3. used ng-repeat and cards to implement information and tested with
hard code successfully.
4. added an Angular module `ng-app="teamform-about-app"
5. added a <div> with ng-controller="AboutCtrl"> to include all the
contents

stevenTCL commented:
Just updated a new commit, plz check it out.

chautsunman commented on app/about.html:
```diff
@@ -0,0 +1,214 @@
+    <script src="js/site.js"></script>
+</head>
+<body>
+    <div ng-app="teamform-about-app" ng-controller="AboutCtrl">
```
move `ng-app` to `<body>`

stevenTCL committed:
indent fixed

1. indentation fixed
2. put <ng-app> to <body>

stevenTCL commented:
fixed
